### PR TITLE
fixed version flags in ui module

### PR DESCRIPTION
**Pull request by sebastianbaginski**
Fixes android build of other project

### CODE REVIEW

File: source/ui/x11/X11Window.ooc [hunk 1]
```diff
@@ -19,7 +19,7 @@ use ooc-ui
 import include/x11
 use ooc-draw
 
-version(unix || apple) {
+version((unix || apple) && !android) {
```
Comment (marcusnaslund) [inline source/ui/x11/X11Window.ooc]: Isn't there a parsing problem with `version` still?

Comment (sebastianbaginski) [inline source/ui/x11/X11Window.ooc]: I think it is like that in Exception class too.

File: source/ui/x11/X11EventLoop.ooc [hunk 2]
```diff
@@ -4,6 +4,7 @@ import include/x11
 import ../[EventLoop, DisplayWindow, GuiEvent]
 import UnixWindow
 
+version(!android && (unix || apple)) {
```
Comment (marcusnaslund) [inline source/ui/x11/X11EventLoop.ooc]: Or does this not parse correctly?

Comment (sebastianbaginski) [inline source/ui/x11/X11EventLoop.ooc]: I have both this and the other version in this PR and it seems to compile for android.

**Comment by sebastianbaginski** [comment]
Updated.

**Pull request merged**